### PR TITLE
feat(cpu): add bfmmla and change sdot to .inst

### PR DESCRIPTION
**Pull request by wangzhaode**
- Add `bfmmla` instruction.
- Change `sdot` to `.inst` to reduce compile dependencies.

### CODE REVIEW

File: src/cpu/aarch64.cpp [hunk 1]
```diff
@@ -232,6 +239,7 @@ void megpeak::aarch64() {
     benchmark(mul_throughput, mul_latency, "mul");
 #if defined(__ARM_FEATURE_MATMUL_INT8)
     benchmark(smmla_throughput, smmla_latency, "smmla", 64);
+    benchmark(bfmmla_throughput, bfmmla_latency, "bfmmla", 64);
```
Comment (chenqy4933) [inline src/cpu/aarch64.cpp]: the operators of bfmmla is 64?

Comment (chenqy4933) [reply src/cpu/aarch64.cpp]: it has 2x4x2 multiplication，and 2x4x2 addition，so it should be 32

Comment (wangzhaode) [reply src/cpu/aarch64.cpp]: yes should be 32

Comment (wangzhaode) [reply src/cpu/aarch64.cpp]: fix it now

**Comment by chenqy4933** [comment]
LGTM

**Pull request merged**